packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-11/wallet-all-ertp/wf-game-propose.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Propose and carry out Wallet Factory upgrade
+
+SDK=${SDK:-/usr/src/agoric-sdk}
+UP11=${UP11:-$SDK/upgrade-test-scripts/agoric-upgrade-11}
+WFUP=${WFUP:-$UP11/wallet-all-ertp}
+
+cd $WFUP
+
+. $SDK/upgrade-test-scripts/env_setup.sh
+. $UP11/env_setup.sh
+
+TITLE="Start Game1 Contract"
+
+DESC="Start Game1 and register well-known Place issuer"
+
+[ -f ./start-game1-permit.json ] || (echo run wf-install-bundle.sh first ; exit 1)
```

Comment:
Without a preceding `set -e` or similar, this `exit 1` in the subshell will not interrupt execution of the containing script.

I can't tell if any of the other shell scripts in this PR are missing `set -e` by mistake.

Reply:
yeah; this is what happens when one is so frustrated that when it stops crashing, one stops coding.
I better write it out long-hand with `if` and such.